content/rest-api/teams.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Teams API
+description: API for managing team members
+
+weight: 5
+---
+
+This document describes the API endpoints for managing teams. To use this API, you must be a **team owner**.
+
+
+>Read more about the [Teams](../getting-started/teams/) feature, the available user roles and permissions.
+
+
+
+## Invite a new team member
+Invite a new team member to your team.
+
+`POST /:team_id/invitation`
+
+#### Parameters
+
+
+| **Name** | **Type** | **Description**                               |
+|----------| -------- |-----------------------------------------------|
+| `email`  | `string` | **Required.** User email                    |
+| `role`   | `string` | **Required.** Could be `owner` or `developer` |

```

Comment:
I would still like to see a note about the developer role corresponding to the member role as seen in the UI.